src/pipelines/question_answering.rs

```diff
@@ -903,6 +903,8 @@ impl QuestionAnsweringModel {
         spans
     }
 
+    /// Pads the passed features to the length of the longest feature. Masks are generated so the
```

Comment:
I totally guessed on this comment about the masks. I probably shouldn't have. Happy to remove/rework those!

Reply:
hi @mlodato517 - This is not entirely accurate, the masks are used to discard any answer generated that would be overlapping with these mask features. This ensure we cannot return as answer containing parts of the question (that is prepended to the context for QA) nor padding.